src/index.ts

```diff
@@ -90,6 +100,44 @@ export default class LogDumpsterPlugin {
     }
   }
 
+  async fetchCurrentStackResources(): Promise<Record<string, string>> {
+    const params = { StackName: this.serviceProvider.stackName }
+    const physicalIds = (await this.provider.request(
+      'CloudFormation',
+      'listStackResources',
+      params
+    )) as unknown as ListStackResourcesResult
```

Comment:
Why the two casts?

Reply:
Without first casting to unknown, I get the following error:
```
Conversion of type 'JSONRepresentable' to type 'ListStackResourcesResult' may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to 'unknown' first.
  Property 'StackResourceSummaries' is missing in type 'JSONRepresentable' but required in type 'ListStackResourcesResult'.
  ```

An alternative solution I just tried that seems to work would be to make `ListStackResourcesResult` and `StackResource` both extend `JSONRepresentable` but neither approach truly guarantees runtime safety

Reply:
It's probably fine. This is pretty low risk. Merge at your leisure.